roles/nginx/files/nginx.nomad

```diff
@@ -129,6 +129,9 @@ job "nginx" {
     {{- else if index .ServiceMeta "socket" -}}
       {{- scratch.Set "registerService" .Name -}}
     {{- end -}}
+    {{- if .Tags | contains "proxy_cache" -}}
+      {{- scratch.Set "setProxy" .Name -}}
+    {{- end -}}
```

Comment:
The existing convention is to use the service metadata to drive Nginx configuration rather than tags.

I also don't think you necessarily need an entire separate indicator on the service for it, you could just look for the presence of `proxy_cache` in the `nginx-config` metadata string.